custom/icds_reports/management/commands/sql_scripts/fix_past_data_part_3.sql

```diff
@@ -0,0 +1,23 @@
+UPDATE "agg_child_health_%(start_date)s_5" agg_child_health
```

Comment:
I don't think this format is valid for all months. At some point last fall we stopped subdividing the monthly agg child health

Reply:
Updated the script to handle both type of tables